src/sass/base/_va.scss

```diff
@@ -1397,37 +1397,41 @@ p.info-message {
 
 .vertical-list-group .list-group-item {
   border: none;
-  border-left: 3px solid $color-gray-light;
+  border-left: 4px solid #cccccc;
   box-sizing: border-box;
   border-radius: 0;
   padding-left: 20px;
   padding-right: 0px;
   padding-bottom: 20px;
   padding-top: 0px;
+  margin-bottom: -30px;
```

Comment:
Looks like `.list-group-item` is used in `claims-appeals.md`, the discharge wizard's guidance page, and the discharge wizard's "request dd214" page. I tested the guidance page and claims-appeals:

### Guidance Page
![image](https://user-images.githubusercontent.com/12970166/37537679-91b6c344-290b-11e8-8e95-d8fe98be1186.png)

### Appeals
![image](https://user-images.githubusercontent.com/12970166/37537717-a72d0fee-290b-11e8-8cc1-72394f6957de.png)

I'm not sure how to get to the dd214 page without digging into it too much, but I'm going to guess it's probably okay there too.

Reply:
please review /discharge-upgrade-instructions/request-dd214 for request-dd214